src/core/src/opsets/opset.cpp

```diff
@@ -6,31 +6,77 @@
 
 #include "ngraph/log.hpp"
 #include "ngraph/ops.hpp"
+#include "openvino/core/node.hpp"
+#include "opset_factory.hpp"
 
 ngraph::OpSet::OpSet(const ov::OpSet& opset) : ov::OpSet(opset) {}
 
+const ngraph::FactoryRegistry<ngraph::Node>& ngraph::OpSet::get_factory_registry() {
+    NGRAPH_SUPPRESS_DEPRECATED_START
+    for (const auto& builder : m_factory->get_builders()) {
```

Comment:
What if I call `get_factory_registry()` several times?
Will it add all builders to this factory several times?

Reply:
As discussed, it will be deprecated. so no need to optimize it